py/picca/pk1d/postproc_pk1d.py

```diff
@@ -737,10 +735,15 @@ def compute_average_pk_redshift(
                 standard_dev, _, _ = binned_statistic(
                     data_snr, p1d_values, statistic="std", bins=snr_bin_edges
                 )
+                standard_dev_error, _, _ = binned_statistic(
+                    data_snr, data_values, statistic="std", bins=snr_bin_edges
+                )
                 standard_dev_full = np.copy(standard_dev)
                 mask = np.isnan(standard_dev)
+                mask |= np.isnan(standard_dev_error)
                 if len(mask[mask]) != 0:
```

Comment:
here a `(~mask).sum()>0` should give the same answer, also typically masks are defined as 0 for throwing away and !=0 for keeping, so maybe define the masks as `~np.isnan`

Reply:
Here I wanted to know if there are any nan, so the equivalent should be mask.sum() > 0 ? I do not know if this is a more optimized version. I mean between a sum and applying a mask, the timing should be similar.

Reply:
Ok I understand your comment know. I will do the change